README.md

```diff
@@ -27,6 +27,8 @@ Board                          |  Connectivity     | Storage for credentials and
 `ST NUCLEO_H743ZI2`                 | Ethernet          | Internal Flash          | Build-only
 `ST NUCLEO_L4R5ZI`                  | Wi-Fi ESP8266     | Internal Flash          | Build-only
 `Nuvoton NUMAKER_IOT_M487`          | Wi-Fi ESP8266     | SD card (NUSD)          | Build-only
+`Renesas RZ_A1H`                    | Ethernet          | External Flash (Note that it is not as secure as internal Flash)         | Build-only
+`Renesas GR_LYCHEE`                    | Wi-Fi (ESP32)  | External Flash (Note that it is not as secure as internal Flash)         | Build-only
```

Comment:
I would want the more explicit statements about production devices using the specially ordered secure versions of the chips being available from Renesas. The board info pages do not mention that;

- https://os.mbed.com/platforms/Renesas-GR-LYCHEE/
- https://os.mbed.com/platforms/Renesas-GR-PEACH/

Might make most sense to have it directly on those pages and just link to them?

Reply:
@JanneKiiskila I added security limitation info for both board page and modified readme.